pattern/constraint/type/PlaysConstraint.java

```diff
@@ -119,13 +119,9 @@ public int hashCode() {
 
     @Override
     public String toString() {
-        StringBuilder syntax = new StringBuilder();
-        syntax.append(PLAYS).append(SPACE);
-        if (relationType != null) syntax.append(relationType.referenceSyntax()).append(COLON);
-        if (roleType != null) syntax.append(roleType.referenceSyntax());
-        if (overriddenRoleType != null)
-            syntax.append(SPACE).append(AS).append(SPACE).append(overriddenRoleType.referenceSyntax());
-        return syntax.toString();
+        return owner.toString() + SPACE + PLAYS
+                + (relation().isPresent() ? "" + SPACE + relationType + COLON : "") + SPACE + roleType.toString()
```

Comment:
I think the spaces are slightly off here, so that we'll get `plays friendship: friend` rather than `plays friendship:friend`. Remove these two and add one following `PLAYS` regardless of whether relation is present

Reply:
Agreed, changed to `plays employment:employee`